mappings/net/minecraft/server/rcon/RconBase.mapping

```diff
@@ -1,7 +1,31 @@
 CLASS net/minecraft/class_3359 net/minecraft/server/rcon/RconBase
 	FIELD field_14423 thread Ljava/lang/Thread;
 	FIELD field_14424 description Ljava/lang/String;
+	FIELD field_14425 server Lnet/minecraft/class_2994;
+	FIELD field_14426 sockets Ljava/util/List;
+	FIELD field_14428 THREAD_COUNTER Ljava/util/concurrent/atomic/AtomicInteger;
+	FIELD field_14429 serverSockets Ljava/util/List;
+	FIELD field_14430 LOGGER Lorg/apache/logging/log4j/Logger;
 	FIELD field_14431 running Z
+	METHOD <init> (Lnet/minecraft/class_2994;Ljava/lang/String;)V
+		ARG 1 server
+		ARG 2 description
+	METHOD method_14721 closeSocket (Ljava/net/ServerSocket;)Z
+		ARG 1 socket
+	METHOD method_14722 log (Ljava/lang/String;)V
```

Comment:
How does this differentiate from other logging methods?

Reply:
Logs a message to the dedicated server hosting the rcon connection